### PR TITLE
Override block properties

### PR DESCRIPTION
**Pull request by itkg-nanne**

### CODE REVIEW

**Comment by nicolasThal** [comment]
:+1:

File: Backoffice/GenerateForm/Strategies/GalleryStrategy.php [hunk 1]
```diff
@@ -44,6 +44,10 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $formats = $this->getFormats();
         $builder
+            ->add('id', 'text', array(
+            'label' => 'open_orchestra_backoffice.form.block.id',
```
Comment (itkg-bfouche) [inline Backoffice/GenerateForm/Strategies/GalleryStrategy.php]: check style

File: Backoffice/GenerateForm/Strategies/GalleryStrategy.php [hunk 2]
```diff
@@ -44,6 +44,10 @@ public function buildForm(FormBuilderInterface $builder, array $options)
...
+            'required' => true
```
Comment (itkg-bfouche) [inline Backoffice/GenerateForm/Strategies/GalleryStrategy.php]: check style

**Comment by itkg-bfouche** [comment]
:+1:

**Comment by itkg-ngilain** [comment]
:+1:

**Pull request merged**